optaplanner-persistence/optaplanner-persistence-jpa/src/main/java/org/optaplanner/persistence/jpa/impl/score/buildin/bendable/BendableScoreConverter.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.persistence.jpa.impl.score.buildin.bendable;
```

Comment:
This should not be in the "impl" package, but the "api" instead.
So `org.optaplanner.persistence.jpa.api.score.buildin.bendable`.

@triceo I 'd even argue we want to drop the ".persistence" part in that url.
So `org.optaplanner.jpa.api.score.buildin.bendable`.

Reply:
@cuijulian For this PR, just put the scores in `org.optaplanner.persistence.jpa.api.score.buildin`
The `.persistence` discussion is out of scope for this PR.

Reply:
Will this also apply to the `*HibernateType` classes?

Reply:
If these classes are becoming a part of our API, let's also make sure we maintain backward compatibility by adding revapi config file like we have for optaplanner-core:
https://github.com/kiegroup/optaplanner/blob/master/optaplanner-core/src/build/revapi-config.json

Reply:
@rsynek Added in latest commit.